Eastern-Research-Group/mywaterway
# Feature/571 custom download widget

## Related Issues:
* [HMW-571](https://jira.epa.gov/browse/HMW-571)

## Main Changes:
* Switched to a custom download widget. This custom download widget has all of the same features of the out of the box widget, but with a simplified UI.
  * The print service fails occasionally (the out of the box widget had the same problem), so I added some retry logic. 

## Steps To Test:
1. Run `npm run ci --legacy-peer-deps`
2. Navigate to http://localhost:3000/community/dc/overview
3. Test various features of the download widget
4. Block the `utility.arcgisonline.com/arcgis/rest/services/Utilities/PrintingTools/GPServer/Export%20Web%20Map%20Task/execute` request
5. Verify the retry logic works 

## TODO:
- [ ]  See if Kiki would like to remove any of the features.
